python/cugraph-service/scripts/default-config.sh

```diff
@@ -12,6 +12,7 @@
 # limitations under the License.
 
 THIS_DIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
+WORKSPACE=$(pwd)/..
```

Comment:
It seems like this could break if run from the wrong directory. Can you make this relative to `THIS_DIR` if possible by default, and also make it override-able by the user if their environment has it defined? 
```suggestion
WORKSPACE=${WORKSPACE:${THIS_DIR}/..}
```

Reply:
I didn't intend to add this change - this should be part of #2977